www/js/transforms/collapseTables.js

```diff
@@ -1,6 +1,6 @@
 var transformer = require("../transformer");
 var utilities = require("../utilities");
-var getTableHeader = require("applib").CollapseElement.getTableHeader;
+var getTableHeader = require("applib").CollapseTable.getTableHeader;
```

Comment:
Here I updated our source to reference the name change from the last commit. I then ran `make grunt` again to regenerate `Wikipedia/assets/index.js`.